packages/atomic/src/components/atomic-result-list/atomic-result-list.pcss

```diff
@@ -1,9 +1 @@
 /* CAREFUL! These styles aren't protected by a shadow DOM. */
-
-atomic-result + atomic-result::before {
-  content: ' ';
-  display: block;
-  height: 2px;
-  margin: 0 2rem;
-  background-color: var(--atomic-neutral);
-}
```

Comment:
Dividers between results were moved back into individual results themselves since results have no divider on mobile and instead have a surrounding border.